src/Goldilocks/Goldilocks.csproj

```diff
@@ -4,6 +4,11 @@
     <TargetFramework>net8.0</TargetFramework>
     <Nullable>enable</Nullable>
     <ImplicitUsings>enable</ImplicitUsings>
+    <NoDefaultLaunchSettingsFile Condition="'$(ExcludeLaunchSettings)' == 'True'">true</NoDefaultLaunchSettingsFile>
+    <RootNamespace>Goldilocks</RootNamespace>
+    <ServerGarbageCollection>False</ServerGarbageCollection>
+    <TrimMode>full</TrimMode>
```

Comment:
Do we pass in the `PublishAot` from the outside? If so, perhaps just pass `TrimMode` too when we pass `PublishAot` given we've removed it from the project file.